packages/bot-web-ui/src/constants/bot-contents.ts

```diff
@@ -2,17 +2,20 @@ type TTabsTitleProps = {
     [key: string]: string | number;
 };
 
+type TDashboardTabsProps = {
+    [key: string]: number;
+};
+
 export const tabs_title: TTabsTitleProps = Object.freeze({
     WORKSPACE: 'Workspace',
     CHART: 'Chart',
 });
 
-export const DASHBOARD_TABS: TTabsTitleProps = Object.freeze({
+export const DASHBOARD_TABS: TDashboardTabsProps = Object.freeze({
```

Comment:
@maryia-matskevich-deriv Can we name it `DBOT_TABS` instead of `DASHBOARD_TABS` because dashboard is also one of the active tabs. The naming of DBOT_TABS may be more accurate.

Reply:
done